syntax = "proto3";

message CreateUser {
    string first_name = 1;
    string last_name = 2;
    string password = 3;
    string email = 4;
}
